README.md

```diff
@@ -67,9 +67,10 @@ Modify the line in `.tmux.conf`, passing the path of the configuration file as a
     gitmux -cfg .gitmux.conf
 
 
-`gitmux` configuration is split into 2 sections:
+`gitmux` configuration is split into 3 sections:
  - symbols: they are just strings of unicode characters
  - styles: they are tmux format strings (`man tmux` for reference)
+ - display: defines the layout in the form of a YAML array, separators should be surrounded in single quotes example: `display: [branch, '..', remote, ' - ', flags]`
```

Comment:
Could we reformulate this a bit?
An example is nice to have but I'd prefer the sentence to be less technical and more in the same tone that the 2 others (symbols & styles)

Reply:
sure thing! work is a little hectic but i should get to this tonight or tomorrow morning

Reply:
how's this:

`gitmux` configuration is split into 3 sections:
 - symbols: they are just strings of unicode characters
 - styles: they are tmux format strings (`man tmux` for reference)
 - display: is the layout of git components 

Example displays: 
```
display: [branch, '..', remote, ' - ', flags]
display: [branch]
display: [flags, ' && ', branch]
```

Reply:
Actually could we call this `layout` instead of `display`?  It seems a better match, as your propositon for the help string pointed out. I mean in the code as well.

So we could go for this:
```
display: this is the layout of git components and separators.
```

No worries for the delay. Thank you to care for this project and for this feature. And sorry for the extra work, the thing is once something gets in master and people start to use it on their config, changing names may break stuff so I prefer to be extra careful about that

Reply:
wait, did you mean:
```
layout: this is the layout of git components and separators.
```
?

Reply:
Yes I totally meant that. I edited my previous reply. Also yes, essentially `s/display/layout`
Sorry for the confusion